ui/src/components/Canvas.tsx

```diff
@@ -1118,24 +1134,24 @@ export function Canvas() {
           multiSelectionKeyCode={isMac ? "Meta" : "Control"}
         >
           <Box>
-            <MiniMap
-              nodeStrokeColor={(n) => {
-                if (n.style?.borderColor) return n.style.borderColor;
-                if (n.type === "code") return "#d6dee6";
-                if (n.type === "scope") return "#f4f6f8";
-
-                return "#d6dee6";
-              }}
-              nodeColor={(n) => {
-                if (n.style?.backgroundColor) return n.style.backgroundColor;
-
-                return "#f4f6f8";
-              }}
-              nodeBorderRadius={2}
-            />
-            <Controls showInteractive={role !== RoleType.GUEST} />
-
-            <Background />
+            {/*<MiniMap*/}
+            {/*  nodeStrokeColor={(n) => {*/}
+            {/*    if (n.style?.borderColor) return n.style.borderColor;*/}
+            {/*    if (n.type === "code") return "#d6dee6";*/}
+            {/*    if (n.type === "scope") return "#f4f6f8";*/}
+
+            {/*    return "#d6dee6";*/}
+            {/*  }}*/}
+            {/*  nodeColor={(n) => {*/}
+            {/*    if (n.style?.backgroundColor) return n.style.backgroundColor;*/}
+
+            {/*    return "#f4f6f8";*/}
+            {/*  }}*/}
+            {/*  nodeBorderRadius={2}*/}
+            {/*/>*/}
+            {/*<Controls showInteractive={role !== RoleType.GUEST} />*/}
+
+            {/*<Background />*/}
```

Comment:
and the controls and the canvas backgrounds?

Reply:
> Thanks, sudongyuer! There appear to be a few coding and design issues.

Yes , so i make this pr to Draft

Reply:
ditto